src/components/MaterialAsyncSelect.js

```diff
@@ -75,8 +75,8 @@ function SelectWrapped(props) {
     styles={customStyles}
     isClearable={true}
     isMulti={props.isMulti}
-    getOptionValue={(option) => { if (props.getOptionValue) { return props.getOptionValue(option) } else { return option.id } }}
-    getOptionLabel={(option) => { if (props.getOptionLabel) {return props.getOptionLabel(option)} else {return option.label}}}
+    getOptionValue={(option) => option.id}
```

Comment:
@barbaraReliefapps Same question here: why did you make these changes on MaterialAsyncSelect ?